protos/storage.proto

```diff
@@ -93,19 +93,19 @@ message WriteResponse {
   uint32 failed = 3;
 }
 
-message QueryRequest {
-  repeated string metrics = 1;
+message SqlQueryRequest {
+  repeated string tables = 1;
   string ql = 2;
 }
 
-message QueryResponse {
+message SqlQueryResponse {
   common.ResponseHeader header = 1;
-  enum SchemaType {
-    AVRO = 0;
-    JSON = 1;
+  uint32 affected_rows = 2;
+  oneof rows_payload {
+    ArrowPayload payload = 3;
   }
-  SchemaType schema_type = 2;
-  string schema_content = 3;
-  repeated bytes rows = 4;
-  uint32 affected_rows = 5;
 }
+
+message ArrowPayload {
+  bytes bytes = 1;
```

Comment:
I guess this field type and name should be:
```suggestion
  repeated bytes record_batches = 1;
```

Reply:
ok, done